### PR TITLE
Css added

### PR DESCRIPTION
**Pull request by Sajawal-dev1**
CSS and empty input box Done

### CODE REVIEW

File: src/AddTodo.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.btn-sub{
```
Comment (alitjanjua) [inline src/AddTodo.css]: name of these kind of files should start with lower case letter.
e.g. `addToDo.css`

File: src/AddTodo.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    background-color: black;
+    color: azure;
+    border: 1px;
+    margin: 4px;
+    padding: 10px;
+    border-style: solid;
+    border-color: aliceblue;
+    position: relative;
+    left:70px
+
```
Comment (alitjanjua) [inline src/AddTodo.css]: remove empty lines

File: src/AddTodo.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    width: 250px;
+    height: 50px;
+    background-color: orange;
+    margin: 0 auto;
+    border: 1px navy solid;  
+    outline: 1px navy solid; 
+    outline-offset: -2px;
+
+     
+  }
```
Comment (alitjanjua) [inline src/AddTodo.css]: add eof here

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+.app-title{
+  margin: 0;
+  margin-bottom: 30px;
+}
+
+.add-todo{
+  margin: 30px 0px;
+}
+
+.task-input{
```
Comment (alitjanjua) [inline src/App.css]: if you've created task related separate css file then you don't have to add that css here too.

File: src/App.css [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+}
+
+.remove-task-button:hover{
+  cursor: pointer;
+}
+.break-line{
+  content: "\a";
+  white-space: pre;
+}
```
Comment (alitjanjua) [inline src/App.css]: add eof here

File: src/App.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import AddTodo from './components/AddTodo';
```
Comment (alitjanjua) [inline src/App.js]: components name should be more meaningful and sensible.
```suggestion
import AddTaskField from './components/AddTaskField';
```

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+import TodoList from './components/TodoList';
```
Comment (alitjanjua) [inline src/App.js]:
```suggestion
import TasksList from './components/TasksList';
```

File: src/App.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+import  "./App.css"
+import "./AddTodo.css"
+const App = () => {
+	return (
+		<div className="app">
```
Comment (alitjanjua) [inline src/App.js]: indentation is not correct.
there should be 2 space indentation enabled.

File: src/components/AddTodo.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { useDispatch } from "react-redux";
+import { addTask } from "../redux/tasksSlice";
```
Comment (alitjanjua) [inline src/components/AddTodo.js]: always use absolute paths
```suggestion
import { addTask } from "store/redux/tasksSlice";
```

File: src/components/AddTodo.js [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+			<input className='task-input'
+				type="text"
+				placeholder="Add task"
+				value={value}
+				onChange={(event) => setValue(event.target.value)}
+			></input>
+			<button  className='task-button'   onClick={onSubmit}>
+				Save
+			</button>
+
```
Comment (alitjanjua) [inline src/components/AddTodo.js]: use empty lines from everywhere

File: src/components/AddTodo.js [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+			addTask({
+				task: value
+			})
+			
+		);
+		setValue(" ");
+	};
+	return (
+		<div className='add-todo'>
+			<input className='task-input'
```
Comment (alitjanjua) [inline src/components/AddTodo.js]: you still haven't used react-hook-form for fields. Do that

File: src/components/TodoItem.js [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+import { deleteTask, editTask } from "../redux/tasksSlice";
+import "../App.css";
+
+const TodoItem = ({ id, title }) => {
+	const [newval, setValue] = useState('');
+	const [check, setcheck] = useState('');
+    const dispatch=useDispatch();
+
+	const edit=()=>{
+		console.log(newval)
```
Comment (alitjanjua) [inline src/components/TodoItem.js]: remove all console statements

File: src/components/TodoList.js [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+				<TodoItem  id={todo.id} title={todo.name}/>
+			))}
+		
+			
+		</ul>
+	
+	);
+};
+
+export default TodoList;
```
Comment (alitjanjua) [inline src/components/TodoList.js]: add eof

File: src/redux/tasksSlice.js [hunk 14]
```diff
@@ -0,0 +1,47 @@
+import { createSlice } from "@reduxjs/toolkit";
+import "../App.css";
```
Comment (alitjanjua) [inline src/redux/tasksSlice.js]: use absolute paths

Comment (alitjanjua) [reply src/redux/tasksSlice.js]:
```suggestion
import "App.css";
```

File: src/redux/tasksSlice.js [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+
+export const tasksSlice = createSlice({
+    name: "tasks",
+    initialState:[],
+    reducers:{
+        
+        addTask: (state, action)=>{
+             let a=0;
```
Comment (alitjanjua) [inline src/redux/tasksSlice.js]: use meaningful names for variables.

**Review by alitjanjua — CHANGES_REQUESTED** [review]

File: src/redux/tasksSlice.js (hunk 14 above)
Comment (alitjanjua) [reply src/redux/tasksSlice.js]: @Sajawal-dev1 this has not been resolved.

**Review by alitjanjua — CHANGES_REQUESTED** [review]

**Comment by Sajawal-dev1** [comment]
No need

**Comment by Sajawal-dev1** [comment]
No more need

**Pull request closed without merging**